crates/bevy_sprite_render/Cargo.toml

```diff
@@ -44,6 +44,9 @@ bitflags = "2.3"
 nonmax = "0.5"
 tracing = { version = "0.1", default-features = false, features = ["std"] }
 
+[dev-dependencies]
+test-case = "3.3.1"
```

Comment:
adds a unique test dependency that isn't used in other crates afaict

Reply:
I can remove it, but it will just make the tests uglier... I'm actually a bit surprised it's not being used elsewhere in the repo...